CIP-Draft-Initial-Params/Initial.md

```diff
@@ -0,0 +1,172 @@
+
+* increasing ``nOpt`` to align more closely with the number of active pools
+* increasing ``a0`` to increase the pledge effect
+* decreasing ``minPoolCost`` (e.g. in line with growth with the Ada value)
+* decreasing ``decentralisationParam`` to 0 (to enable full decentralisation of block production)
+
+Further adjustments are likely to be required to tune the system as it evolves.
+
+
+### Economic Parameters
+
+Four parameters govern the economics of the system:  ``tau``, ``rho``, ``minFeeA`` and ``minFeeB``.
+The first two concern the rate of rewards that are provided to stake pools, delegators and the treasury.
+The others concern transaction costs.
+
+
+### Transaction and Block Sizes
+
+Three parameters govern block and transaction sizes: ``maxBlockBodySize``, ``maxBlockHeaderSize``, ``maxTxSize``.
+Their settings have been chosen to ensure the required levels of functionality, within
+constrained resource restrictions (including long-term blockchain size and real-time worldwide exchange of blocks).
+Changes to these parameters may impact functionality, network reliability and performance.
+
+
+## Backward Compatibility
+
+This CIP describes the initial set of protocol parameters, so backwards compatibility is not an issue.
+Future CIPs may be proposed to change any or all of these parameters.
+A change to the major protocol version indicates a major change in the node software.
+Such a change may involve adding/removing parameters or changing their semantics/formats.
+In contrast, minor protocol changes are used to ensure key software updates without changing
+the meaning of protocol parameters.
+
+
+## Change Process for Protocol Parameter Changes
+
+### Governance
+
+Changes will affect many stakeholders and must therefore be subject to open community debate and discussion.
+
+Ultimately, the Voltaire protocol voting mechanism will be used to achieve fully automated, decentralised and transparent governance.
+In the interim, the CIP process will be used.
+
+### Signalling Protocol Parameter Changes
+
+Changes to the parameters need to be signalled to the community well in advance, so that they can take appropriate action.  For the most significant parameters, a minimum of 4-6 weeks
+elapsed time between announcement and enactment is appropriate.  This period must be included in the CIP.  Announcements will be made as soon
+as practical after the conclusion of the vote.
+
+### Applying Protocol Parameter Changes
+
+Protocol parameter changes must be submitted within the first 24 hours of the epoch before they are required to come into effect.
+Once a change has been submitted, it cannot be revoked.
+
+### Voiding Proposed Protocol Parameter Changes
+
+Once a protocol parameter change has been announced, it can only be overridden through the voting process (CIP, Voltaire etc.).  Any vote must be
+completed before the start of the epoch in which the change is to be submitted.
+
```

Comment:
What about adding a changelog at the end of this CIP with updates done on the various initial parameters? Either linking to another CIP (explaining rationale, motivation and the nature of the change) or with a short sentence explaining the rationale.

Reply:
Yes, that would be the plan.  I wouldn't want a CIP for each of the d parameter changes that have happened so far, though - probably best to roll them into a single CIP.

Reply:
Added a section describing changes.